cla_socketserver/admin.js

```diff
@@ -21,42 +21,6 @@
       peopleMap: function (req, res) {
```

Comment:
Do we need this function now?

Reply:
Nope, don't need it.
Good spot, i've removed it now